src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1269,26 +1273,25 @@ public void run() {
 	}
 
 	/*
-	 * This disables fall damage whilst driving a car
+	 * This disables minor fall damage whilst driving a car
 	 */
 	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
 	void safeFly(EntityDamageEvent event) {
 		if (!(event.getEntity() instanceof Player) || event.getCause() != DamageCause.FALL) {
 			return;
 		}
 		Player p = (Player) event.getEntity();
-		if (inACar(p.getName())) {
-			event.setCancelled(true);
-			/*Vector vel = p.getVehicle().getVelocity();
-			if (!(vel.getY() > -0.1 && vel.getY() < 0.1)) {
-			} else {
+		if (inACar(p.getName()) && !disableFallDamage) {
+			Vector vel = p.getVehicle().getVelocity();
+			if (vel.getY() > -0.1 && vel.getY() < 0.1) {
+				event.setCancelled(true);
+			}/*else {
```

Comment:
Surely this is missing an "if(disableFallDamage) { cancel event}"?

Reply:
Don't know how I missed that